<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagListEditor</class>
 <widget class="QWidget" name="TagListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="UniqueEditableListView" name="tag_list_view">
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="edit_buttons" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="tags_add_btn">
          <property name="text">
           <string>Add new tag</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="sort_buttons" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="tags_move_up_btn">
             <property name="toolTip">
              <string>Move tag up</string>
             </property>
             <property name="accessibleName">
              <string>Move tag up</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme=":/images/16x16/go-up.png">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tags_move_down_btn">
             <property name="toolTip">
              <string>Move tag down</string>
             </property>
             <property name="accessibleName">
              <string>Move tag down</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme=":/images/16x16/go-down.png">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tags_remove_btn">
          <property name="toolTip">
           <string>Remove selected tags</string>
          </property>
          <property name="accessibleName">
           <string>Remove selected tags</string>
          </property>
          <property name="text">
           <string>Remove tags</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UniqueEditableListView</class>
   <extends>QListView</extends>
   <header>picard.ui.widgets.editablelistview</header>
   <slots>
    <slot>add_empty_row()</slot>
    <slot>remove_selected_rows()</slot>
    <slot>move_selected_rows_up()</slot>
    <slot>move_selected_rows_down()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tags_add_btn</sender>
   <signal>clicked()</signal>
   <receiver>tag_list_view</receiver>
   <slot>add_empty_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tags_remove_btn</sender>
   <signal>clicked()</signal>
   <receiver>tag_list_view</receiver>
   <slot>remove_selected_rows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tags_move_up_btn</sender>
   <signal>clicked()</signal>
   <receiver>tag_list_view</receiver>
   <slot>move_selected_rows_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tags_move_down_btn</sender>
   <signal>clicked()</signal>
   <receiver>tag_list_view</receiver>
   <slot>move_selected_rows_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
